app/controllers/import_controller.rb

```diff
@@ -278,6 +279,17 @@ def import_date_field(value)
     value.blank? ? nil : value
   end
 
+  def import_time_field(field, value, row_ind)
+    return nil if value.blank?
+
+    normalized_value = value.to_s.downcase.strip.gsub(/[ -.]/, '')
```

Comment:
Where would the gsub come into play here? This lets me import values like `10:00---`, which isn't necessarily a big deal, since no one would do that without knowing the code, but I'm just not seeing why allow it.

Reply:
that's true, I kind of just copied the formatting that was being done in the other normalize method but that's not necessary so I'll remove it

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1019/commits/daa0195b5441006dc8259d33d13da45f325d66c6#diff-ee31e076a80cc9f48da0394c0cb066d59bf2f8c68f68eb62cb5119d01712d90eR285